units/XNA0303/XNA0303_unit.bp

```diff
@@ -255,17 +256,17 @@ UnitBlueprint {
         GroundCollisionOffset = 2,
         MaxSpeed = 0.5,
         MeshExtentsX = 2,
-        MeshExtentsY = 0.55,
-        MeshExtentsZ = 1.65,
+        MeshExtentsY = 0.5,
+        MeshExtentsZ = 2.0,
         MotionType = 'RULEUMT_Air',
     },
     SelectionSizeX = 1,
     SelectionSizeZ = 1.35,
-    SelectionThickness = 0.35,
+    SelectionThickness = 0.36,
```

Comment:
visual change - doesnt need to be changed, unless you specifically think that its got a better selection box this way